templates/neutronapi/config/neutron-api-config.json

```diff
@@ -1,12 +1,18 @@
 {
-  "command": "/usr/bin/neutron-server --config-file /usr/share/neutron/neutron-dist.conf --config-file /etc/neutron/neutron.conf",
+  "command": "/usr/bin/neutron-server --config-file /usr/share/neutron/neutron-dist.conf --config-file /etc/neutron/neutron.conf --config-dir /etc/neutron/neutron.conf.d",
```

Comment:
Does the combination of `--config-file` and `--config-dir` work appropriately?  I know that for Glance, for instance, you can't use them together (one ends up being ignored).  I need a quick search for Neutron docs in this regard, but couldn't find anything.

Reply:
From my tests using both together worked fine, from functionality wise i think it should be same for both glance and neutron as both using oslo.config.

Reply:
Okay, just wanted to check.  With regards to Glance, I'll just point this out:
```
If --config-dir is set, then --config-file is ignored.
```
That's from this doc: https://docs.openstack.org/glance/latest/configuration/configuring.html